_includes/silicon-valley/_interaction_area.html

```diff
@@ -1,5 +1,5 @@
 <div class="experience-buttons">
-  <a class="button {% if page.section == "slideshow" %}active{% endif %}">A history in 22 objects</a>
+  <a class="button {% if page.section == "slideshow" %}active{% endif %}">A history in 22&nbsp;objects</a>
```

Comment:
huh. Is this encoded space needed?

Reply:
I added it to avoid orphaning "objects" when the button text wraps.